src/core/driver/driver.ml

```diff
@@ -105,7 +105,7 @@ let scope_matches { includes; ignores } file =
 exception Break of Neal.Rule.ruleset list
 let collect_rules file rules = function
   | (`Global, rs) -> rs @ rules
-  | (`Scoped scope, rs) when scope_matches scope file -> raise (Break rs)
+  | (`Scoped scope, rs) when scope_matches scope file -> rs @ rules
```

Comment:
@sm11963 can you help review this change? I have no OCaml expertise.

Reply:
Sorry, some changes are still pending. I'll request a review after that.